cmd/windows_run_test.bat

```diff
@@ -0,0 +1,21 @@
+@echo off
+call cmd\getopt.bat %*
+
+if "%PHP_VER%" == "7.4.33" (
+    set CRT=vc15
+) else (
+    set CRT=vs16
+)
+
+if NOT DEFINED ARCH set ARCH=x64
```

Comment:
😂 I had no idea this was the syntax for optional environment vars... I Hate windows so much.

Reply:
That's the whole issue to begin with. You don't do Windows or like, and trying to understand what's happening. 

It might be best you review there official building steps and look at the scripts they have in there.

Reply:
sorry, but compared to the syntax I am used to which works on multiple operating systems, is much shorter, and has more developer mindshare.
https://stackoverflow.com/a/2013589/1548557

Also this syntax is not present in the existing PHP build system.

I'm not evaluating this simply based upon familiarity. If it is part of a build system used by PHP; I'd like it to at least be internally similar and work towards other mechanisms.

Also while I joked, I added a ticket to my own investigation to see if this syntax could make things more readable. You don't need to defend Windows, it has billions of dollars being wasted on it's promotion and development.

Reply:
It's doesn't matter about your jokes. Most PHP only programmers have that problem. It just seem you trying to come up with a solution when you really don't know Windows at all. 

Read there current official Windows process that I changed and the reason it works in a single step. You reading scripts with no Windows background and making judgement callsthen asking at the same time.

Reply:
I'm not sure why you still replying/comments on this PR. 

I can't help you. Read https://wiki.php.net/internals/windows/stepbystepbuild which is mentioned in this PR. Download those scripts review them, then submit PR to original author. 

You should actually read my first post of PR and the reply that went south the moment he mentions about Windows.